### PR TITLE
add tone with npm auto-update

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8475 
@pvnr0082t, @x09326 please help me review this pull request, thank you.
Manually add the old versions because v0.0.1 has different folder pattern.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Tonejs/Tone.js
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/tone
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1743
   - Count of watchers: 85
   - Count of forks: 141
 * NPM download stats (optional):
   - Downloads in the last day: 2
   - Downloads in the last week: 55
   - Downloads in the last month: 299

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by pvnr0082t** [comment]
@kennynaoh There is already a PR for `Tone.js`. #4811 What should we do?@PeterDaveHello

**Comment by PeterDaveHello** [comment]
Please close the PR in the commit message at the same time.

**Comment by kennynaoh** [comment]
@pvnr0082t , @x09326 ,
I have closed the PR #4811 in the commit message.
Please help me review it again.
Thanks!

**Comment by x09326** [comment]
I think [this](https://tonejs.github.io/docs/) can be took as ```homepage```.

**Comment by PeterDaveHello** [comment]
Maybe just https://tonejs.github.io/ ?  Though it'll just redirect to its GitHub repo page right now.

BTW, please just past the url directly, so that it's easier to read it, thanks.

**Comment by kennynaoh** [comment]
@PeterDaveHello , OK, thanks!

**Comment by kennynaoh** [comment]
@pvnr0082t , @x09326 ,
I have fixed the `homepage` field in `package.json`.
Please help me review it again.
Thanks!

File: ajax/libs/tone/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "Synthesis",
+    "Playback",
+    "Effect",
+    "Instrument",
+    "DSP",
+    "Signal Processing",
+    "Interactive Music"
+  ],
+  "author": "Yotam Mann <yotammann@gmail.com> (http://yotammann.info/)",
+  "hpmepage": "https://tonejs.github.io/",
```
Comment (pvnr0082t) [inline ajax/libs/tone/package.json]: There's a typo `"homepage"`.

Comment (kennynaoh) [inline ajax/libs/tone/package.json]: It will be `homepage` in the future.

Comment (PeterDaveHello) [inline ajax/libs/tone/package.json]: Can't be fixed now?

Comment (kennynaoh) [inline ajax/libs/tone/package.json]: Sorry, I misunderstood.
I can fix it now.

**Comment by kennynaoh** [comment]
@pvnr0082t , @x09326 ,
I have fixed `homepage` field.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by x09326** [comment]
@kennynaoh 
Please rebase it.

**Comment by kennynaoh** [comment]
@pvnr0082t , @x09326 
I have rebased to the latest master branch.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by x09326** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@x09326 @pvnr0082t please merge this, thanks.

**Pull request merged**